Appraisals

```diff
@@ -9,11 +9,13 @@ appraise "rails-4-2" do
   gem "test-unit", "~> 3.0"
   gem "sqlite3", "1.3.13"
   gem "rails-controller-testing"
+  gem 'responders', '~> 2.0'
```

Comment:
it was required because I'd use respond_to :all outside the methods, but it didn't work as expected. I'm removing it